jobs/ci-kubernetes-e2e-gke-container_vm-1.3-container_vm-latest-upgrade-cluster-new.env

```diff
@@ -0,0 +1,13 @@
+### job-env
+
+E2E_OPT=--check_version_skew=false
+E2E_UPGRADE_TEST=true
+GINKGO_UPGRADE_TEST_ARGS=--ginkgo.focus=\[Feature:ClusterUpgrade\] --upgrade-target=ci/latest-latest --upgrade-image=container_vm
+JENKINS_PUBLISHED_SKEW_VERSION=ci/latest-latest
```

Comment:
Oh geeze it was not very consistent from the beginning: [here](https://github.com/kubernetes/test-infra/blob/286075f1c13e88e0f702ceaccd587cf23a9bdcf5/jenkins/job-configs/kubernetes-jenkins/kubernetes-upgrade.yaml#L428) and [there](https://github.com/kubernetes/test-infra/blob/286075f1c13e88e0f702ceaccd587cf23a9bdcf5/jenkins/job-configs/kubernetes-jenkins/kubernetes-upgrade.yaml#L180)

@xiangpengzhao I'm happy with just leave `--upgrade-target=ci/latest-latest`